### PR TITLE
[IMP] crm: tips fire proper place as well as when drag tip automaticall ...

### PR DESCRIPTION
**Pull request by ara-odoo**
...discard.
https://www.odoo.com/web#id=10276&view_type=form&model=project.task&menu_id=3942&action=327
pad: https://pad.odoo.com/p/r.emkiiDBErZm9aGpK

### CODE REVIEW

**Comment by rim-odoo** [comment]
@jpa-odoo Can you rebase and fix the conflicts? Thanks

**Comment by jpa-odoo** [comment]
Hello @rim-odoo 
          Conflict are solved, and branch re-based and pushed on github, thanks

File: addons/web_tip/static/src/js/tip.js [hunk 1]
```diff
@@ -250,6 +250,10 @@ var Tip = Class.extend({
                     def.resolve();
                 }
             });
+            self.$element.on('mousedown', function($ev) {
+                self.end_tip(tip);
+                def.resolve();
+            });
```
Comment (rim-odoo) [inline addons/web_tip/static/src/js/tip.js]: Not sure this code is needed, but sure that you should probably update tip `crm_tip_4` and make it work like `project_tip_2`

Comment (jpa-odoo) [inline addons/web_tip/static/src/js/tip.js]: Yes, we can archive crm tip same as like <code>project_tip_2</code> way..

Comment (rim-odoo) [inline addons/web_tip/static/src/js/tip.js]: @jpa-odoo Don't do anything, I did it for you. But recall that it is better to try to fix the data BEFORE thinking to change the generic framework

**Pull request merged**